priv/static/swagger_v2.yaml

```diff
@@ -685,6 +891,296 @@ components:
+      operationId: get_aex9_recipient_transfers
+      parameters:
+        - description: Recipient account id
+          in: path
+          name: id
+          required: true
+          type: string
+          x-example: ak_2meHkLcAoZPrQj7P5WjFyJJRLJqRtv43z1QEbpcS1gHs9W8Q3g
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Returns paginated AEX-9 transfers of a recipient
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - type: object
+                    properties:
+                      data:
+                        type: array
+                        items: 
+                          $ref: '#/components/schemas/Aex9TransferResponse'
+                  - $ref: '#/components/schemas/PaginatedResponse'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+  /aex9/transfers/from-to/{sender}/{recipient}:
+    get:
+      deprecated: false
+      description: Get AEX9 transfers filtered by sender and recipient.
+      operationId: get_aex9_pair_transfers
+      parameters:
+        - description: Sender account id
+          in: path
+          name: recipient
+          required: true
+          type: string
+          x-example: ak_17qFR9Zp6CbX9ZZrEYd5155KHGAsfbY1YZ2UsRpjH6WJc8Pk1
+        - description: Recipient account id
+          in: path
+          name: recipient
+          required: true
+          type: string
+          x-example: ak_psy8tRXPzGxh6975H7K6XQcMFVsdrxJMt7YkzMY8oUTevutzw
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Returns paginated AEX-9 transfers between two accounts
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - type: object
+                    properties:
+                      data:
+                        type: array
+                        items: 
+                          $ref: '#/components/schemas/Aex9TransferResponse'
+                  - $ref: '#/components/schemas/PaginatedResponse'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+  /aex9/{:contract_id}/balances/{:account_id}/history:
+    get:
+      deprecated: false
+      description: Get AEX9 account balance on a contract throughout all heights when changed.
+      operationId: get_aex9_contract_account_balance_history
+      parameters:
+        - description: Contract id
+          in: path
+          name: contract_id
+          required: true
+          type: string
+          x-example: ct_uGk1rkSdccPKXLzS259vdrJGTWAY9sfgVYspv6QYomxvWZWBM
+        - description: Account id
+          in: path
+          name: account_id
+          required: true
+          type: string
+          x-example: ak_152krtQsp2T3ApcFv7B21FFXkjXnsps3VBfhJeyNF1tHh6mB2
```

Comment:
I believe since this are mostly paginated responses they should include at least some of the parameters 
```
  - $ref: '#/components/parameters/LimitParam'
  - $ref: '#/components/parameters/ScopeParam'
  - $ref: '#/components/parameters/DirectionParam'
```

not sure which, it would depend on the endpoint, not all endpoints support all of these parameters

this applies to the rest of the aex9 paginated responses as well